rts/Game/UI/MouseHandler.cpp

```diff
@@ -226,7 +226,8 @@ void CMouseHandler::MouseMove(int x, int y, int dx, int dy)
 {
 	// FIXME: don't update if locked?
 	lastx = std::abs(x);
```

Comment:
I'm considering an approach to sanitize internal mouse coordinates when off-screen. Honestly `abs(coords)` does not seem suitable and we'd likely want to stay out of the border to not pan the screen indefinitely so vertices and borders should be avoided as well. Probably `viewCenter` is the best candidate.